lua/utils/init.lua

```diff
@@ -99,9 +100,10 @@ function utils.reload_lv_config()
   vim.cmd ":PackerCompile"
   vim.cmd ":PackerInstall"
   -- vim.cmd ":PackerClean"
+  vim.g.colors_name = lvim.colorscheme
+  vim.cmd("colorscheme " .. lvim.colorscheme)
```

Comment:
do we need both of these, i think we had some issue, and added the extra one, but never checked again if we actually need both or not

Reply:
it might be better to deal with this in another PR